packages/one-app-bundler/bin/webpackCallback.js

```diff
@@ -15,9 +15,11 @@
 import chalk from 'chalk';
 import fs from 'node:fs';
 import path from 'node:path';
+import snakeCase from 'lodash.snakecase';
```

Comment:
Can we just implement this here instead of adding a dependency (esp lodash)

```js
function toSnakeCase(str) {
  return str
    .replace(/[\s-]+/g, '_')    // Replace spaces and dashes with _
    .replace(/([A-Z])/g, '_$1') // Prefix uppercase letters with _
    .toLowerCase()              // Convert the whole string to lowercase
    .replace(/^_/, '');         // Remove leading underscore if present
}
```

Reply:
We already have all of `lodash` we don't need `lodash.snakecase` specifically as well right?